### PR TITLE
fixing DAS-66: Cassandra data migration not working for large data set

### PR DESCRIPTION
**Pull request by swsachith**
This PR solves the merge conflicts of the previous one.

### CODE REVIEW

File: components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java [hunk 1]
```diff
@@ -72,37 +71,57 @@
...
+    private static final String LONG_TYPE = "org.apache.cassandra.db.marshal.LongType";
+    private static final String INTEGER_TYPE = "org.apache.cassandra.db.marshal.IntegerType";
+    private static final String TIME_UUID_TYPE = "org.apache.cassandra.db.marshal.TimeUUIDType";
+    private static final String ASCII_TYPE = "org.apache.cassandra.db.marshal.AsciiType";
+    private static final String BYTE_TYPE = "org.apache.cassandra.db.marshal.ByteType";
+    private static final String BOOLEAN_TYPE = "org.apache.cassandra.db.marshal.BooleanType";
+    private static final String FLOAT_TYPE = "org.apache.cassandra.db.marshal.FloatType";
+    private static final String DOUBLE_TYPE = "org.apache.cassandra.db.marshal.DoubleType";
 
     public static void main(String[] args) throws Exception {
```
Comment (sinthuja) [inline components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java]: This method is too long, please break into logical methods.

File: components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java [hunk 2]
```diff
@@ -188,54 +252,61 @@ public static void main(String[] args) throws Exception {
             if (service != null) {
```
Comment (sinthuja) [inline components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java]: Closing operations should be done in the finally block.

File: components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java [hunk 3]
```diff
@@ -188,54 +252,61 @@ public static void main(String[] args) throws Exception {
...
-                    break;
-                case BIGDECIMAL:
-                    values.put(columnName, row.getDecimal(originalColumnName).doubleValue());
-                    break;
-                default:
-                    values.put(columnName, row.getString(originalColumnName));
-                    break;
-            }
+    /**
+     * Un-marshal the cassandra returned byte array into Object.
```
Comment (sinthuja) [inline components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java]: Java comments needs to be added to the parameters.

File: components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java (hunk 1 above)
Comment (swsachith) [inline components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java]: fixed with https://github.com/wso2/carbon-analytics/pull/137

File: components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java (hunk 2 above)
Comment (swsachith) [inline components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java]: This has already been fixed.

File: components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java (hunk 3 above)
Comment (swsachith) [inline components/analytics-tools/org.wso2.carbon.analytics.tools.migration/src/main/java/org/wso2/carbon/analytics/tools/migration/AnalyticsDataMigrationTool.java]: fixed with https://github.com/wso2/carbon-analytics/pull/137

**Pull request merged**